models/articles.models.js

```diff
@@ -26,3 +27,29 @@ exports.fetchArticleById = (article_id) => {
     }
   });
 };
+
+exports.fetchArticleComments = (article_id) => {
+  const queryString = `SELECT * FROM comments WHERE article_id = $1`;
+  return this.fetchArticleById(article_id)
+    .then(() => {
+      return db.query(queryString, [article_id]);
+    })
+    .then((result) => result.rows);
+};
+
+exports.addArticleComment = (article_id, body) => {
+  if (!body.username || !body.body)
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  return this.fetchArticleById(article_id)
+    .then(() => checkUserExists(body.username))
+    .then(() => {
+      const queryString = `
+        INSERT INTO comments (article_id, author, body, votes)
+        VALUES ($1, $2, $3, 0)
```

Comment:
Votes will default to 0 when a comment is inserted (👀 have look at the seed function to confirm this) so no need to include in the values here.

Reply:
Fixed (votes and 0 value removed)